cmd/v/internal/help/run.txt

```diff
@@ -0,0 +1,12 @@
+Usage: v [build flags] run <file.v|directory> [arguments...]
+
+Run compiles and runs the named target.
```

Comment:
`Compiles, then runs the file or module.` ?

`Run compiles and runs` seems ambiguous and slightly repetitive to me.